marketplace/enrich.schema.json

```diff
@@ -9,24 +9,193 @@
             "additionalProperties": false,
             "properties": {
                 "weight": {
-                    "type": "boolean",
-                    "default": false
+                    "type": "object",
+                    "additionalProperties": false,
+                    "description": "Enabled/disable weight enricher",
```

Comment:
IMO, Enricher parameters description should provide general information about the operation that is done on the data, not a message indicating that it is possible to activate or desactivate the operation. ;)

for this specific operation I'd say something like : _"Try to fill in vehicle weight if not provided by the seller"_

Same comment for other params. :)